### ISSUE TITLE
"+" in Query parameters

### ISSUE THREAD

**Issue by GFunkloostar** [joeldev/JLRoutes#87]
I have a deeplink where the URL uses the "+" symbol to indicate spaces. Not only in the path, but also in the query part.

For example:
...myBaseURL/invitations/hXVzNLTySmb4TGDL2yxR/accept?invitation_email=daan%40example.com&list_name=Test2+%F0%9F%98%82+%2BHans&actor_name=Daan+De+Boer

Now the NSDictionary *parameters in the handler still contain those  "+" symbols instead of being replaced with spaces. So I added this code in my application:continueUserActivity:restorationHandler: method:

`NSString *changedURL = [[userActivity.webpageURL absoluteString] stringByReplacingOccurrencesOfString:@"+" withString:@"%20"];
NSURL *outputURL = [NSURL URLWithString:changedURL];
return [JLRoutes routeURL:outputURL];`

In my opinion JLRoutes should take care of this, or am I missing something?

**Comment by joeldev** [comment]
JLRoutes has a global option already for this, but it was not being properly applied to query params:
```objc
/// Tells JLRoutes that it should manually replace '+' in parsed values to ' '. Defaults to YES.
+ (void)setShouldDecodePlusSymbols:(BOOL)shouldDecode;
```

Good find!

**Comment by joeldev** [comment]
This fix is now available in JLRoutes 2.0.2 (just pushed to CocoaPods).
